### PR TITLE
[Video] use correct max size

### PR DESCRIPTION
**Pull request by mozzius**

### CODE REVIEW

**Comment by surfdude29** [comment]
Does this string need to be updated too?
https://github.com/bluesky-social/social-app/blob/723a5e488eab40eac63a1983fc8cbbfa3992b3e7/src/state/queries/video/video.ts#L227

File: src/lib/media/video/errors.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export class VideoTooLargeError extends Error {
   constructor() {
-    super('Videos cannot be larger than 100MB')
+    super('Videos cannot be larger than 50mb')
```
Comment (surfdude29) [inline src/lib/media/video/errors.ts:3]:
```suggestion
    super('Videos cannot be larger than 50MB')
```
nit: I think the capitalised MB was correct as it's megabytes being referred to?

Comment (haileyok) [reply src/lib/media/video/errors.ts:3]: Good call, lets do that.

**Review by haileyok — APPROVED** [review]

File: src/lib/media/video/errors.ts (hunk 1 above)
Comment (haileyok) [reply src/lib/media/video/errors.ts:3]: (thank you!!)

Comment (haileyok) [reply src/lib/media/video/errors.ts:3]: Ffs, it didn't capitalize it lol. Opening a PR.

Comment (haileyok) [reply src/lib/media/video/errors.ts:3]: Ahh, actually doesn't matter. That string doesn't get displayed to the user anyway.

**Pull request merged**